src/Controller/GameController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Board;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+
+class GameController extends AbstractController
```

Comment:
globally, this is not a "real" controller, in the symfony MVC sense ... We are more on a business object managing the game logic, than on a controller managing the http request/response...